### PR TITLE
Hovering

### PR DESCRIPTION
**Pull request by cmonez**
Add hovering to photo grid, added button to bottom right corner of grid. Grabbed photo using params @samasastudio @krisdo @andyoe0102

### CODE REVIEW

File: client/components/PhotogridComponents/Photogrid.jsx [hunk 1]
```diff
@@ -20,6 +20,11 @@ const Big = styled.div`
   grid-area: 1 / 1 / 3 / 3;
   border-top-left-radius: 15px;
   border-bottom-left-radius: 15px;
+  &:hover {
+    -webkit-filter: grayscale(50%);
```
Comment (krisdo) [inline client/components/PhotogridComponents/Photogrid.jsx:24]: oh I am interested in what this does

File: client/components/PhotogridComponents/Photogrid.jsx [hunk 2]
```diff
@@ -20,6 +20,11 @@ const Big = styled.div`
...
+    filter: brightness(80%);
+    opacity: 0.8;
+  }
 `;
 
 const Photo1 = styled.div`
```
Comment (krisdo) [inline client/components/PhotogridComponents/Photogrid.jsx:30]: when you have time, could you think of a way to refactor and write less code? It seems as though it's redundant. However, great on organizing your styling into different component and making it easy to read and follow

Comment (cmonez) [reply client/components/PhotogridComponents/Photogrid.jsx:30]: Yeah i def want to! Isn't very dry rn. Just gotta figure out a way to have each grid elemenet be positioned correctly using props.

File: client/components/PhotogridComponents/Photogrid.jsx (hunk 1 above)
Comment (cmonez) [reply client/components/PhotogridComponents/Photogrid.jsx:24]: It greys out something when hovering it!

**Pull request merged**